spec/parallel_spec.rb

```diff
@@ -188,7 +188,7 @@ def cpus
 
     it 'does not leave processes behind while running' do
       skip if ENV['TRAVIS'] # this randomly fails on travis all the time :(
-      `ruby spec/cases/closes_processes_at_runtime.rb`.should == 'OK'
+      `ruby spec/cases/closes_processes_at_runtime.rb`.split(/\n/).last.should == 'OK'
```

Comment:
this is a jruby fix ... but jruby got removed ?

Reply:
It's for ruby 2.6's BigDecimal warnining https://travis-ci.org/grosser/parallel/jobs/495262000

Reply:
would prefer to ignore that explicitly then
`.sub(/.*use BigDecimal.*\n/, "")`

Reply:
fixed